mujoco_warp/_src/test_util.py

```diff
@@ -94,18 +96,26 @@ def fixture(
       mjm.opt.jacobian = mujoco.mjtJacobian.mjJAC_SPARSE
     else:
       mjm.opt.jacobian = mujoco.mjtJacobian.mjJAC_DENSE
+
   mjd = mujoco.MjData(mjm)
   if keyframe > -1:
     mujoco.mj_resetDataKeyframe(mjm, mjd, keyframe)
+
   if kick:
     # give the system a little kick to ensure we have non-identity rotations
     mjd.qvel = np.random.uniform(-0.01, 0.01, mjm.nv)
     mjd.ctrl = np.random.normal(scale=0.01, size=mjm.nu)
     mujoco.mj_step(mjm, mjd, 3)  # let dynamics get state significantly non-zero
+
+  if mjm.nmocap:
+    mjd.mocap_pos = np.random.random(mjd.mocap_pos.shape)
+    mjd.mocap_quat = np.random.random(mjd.mocap_quat.shape)
```

Comment:
should we normalize `np.random.random(mjd.mocap_quat.shape)` so that `mjd.mocap_quat` contains unit quaternions?